src/scripts/main.js

```diff
@@ -1,3 +1,17 @@
 'use strict';
 
-// write your code here
+const totalPop = document.getElementsByClassName('total-population');
```

Comment:
```suggestion
const totalPop = document.querySelector('.total-population');
```
Такой элемент один, зачем искать его через Elements, чтобы потом еще доставать нулевой элемент из массива?